configure.ac

```diff
@@ -301,6 +301,8 @@ AC_SYS_LARGEFILE
 AC_CHECK_PROGS([perl_found], [perl], [yes], [no])
 AM_CONDITIONAL([RUN_OPTLIB2C], [test "${perl_found}" = "yes"])
 
+AC_CHECK_PROGS([rst2man_found], [rst2man], [yes], [no])
```

Comment:
According to [the docs](https://www.gnu.org/software/autoconf/manual/autoconf-2.68/html_node/Generic-Programs.html) the 3rd parameter is `value-if-not-found` and the 4th is the search path, unlike the singular `AC_CHECK_PROG` macro.

I usually do like this:

```
AC_PATH_PROG([RST2MAN], [rst2man], [no])
AM_CONDITIONAL([HAVE_RST2MAN], [test "x$RST2MAN" != "xno"])
```

Then in the `Makefile.am` you can call `$(RST2MAN)` to use the same binary Autoconf found, or even use `$(AM_V_GEN)$(RST2MAN)` if you want to also support silent-rules.